Trader.Client/Views/LogEntryViewer.cs

```diff
@@ -99,11 +99,11 @@ public LogEntryViewer(ILogEntryService logEntryService)
 
```

Comment:
I am writing a comment to explain the workflow. In this instance the change is simple and I will accept it as soon as you respond to this comment.  However, if there are any questions or changes to be made this is the opportunity for the repo owner to make suggestions or demand code changes. 

If you had to change the code, then it is simple. You change the code on your local version of the  repo, check it in, push it and as if by magic the change is reflected here - easy isn't it?